tests/mfx_dispatch/linux/mfx_dispatch_test_cases_plugins.cpp

```diff
@@ -41,6 +41,22 @@ TEST_F(DispatcherPluginsTest, ShouldSearchForPluginsCfgByCorrectPath)
     MFXVideoUSER_Load(session, &uid, 0);
 }
 
+TEST_F(DispatcherPluginsTest, ShouldFailIfPluginIsNotFound)
+{
+    MockCallObj& mock = *g_call_obj_ptr;
+
+    std::string path = std::string(MFX_PLUGINS_CONF_DIR) + "/plugins.cfg";
+    EXPECT_CALL(mock, fopen(StrEq(path.c_str()), _)).Times(1).WillRepeatedly(Return(MOCK_FILE_DESCRIPTOR));;
```

Comment:
looks like, you don't need ```std::string``` at all, ```MFX_PLUGINS_CONF_DIR "/plugins.cfg"``` is already a C-string which you can pass directly to ```fopen```